jeremywiebe/terraform
# Web 2067 2

There were some NPM vulnerabilities shown in the documentation-theme project which was then traced back to here. Thus, some dependencies were added and/or updated.

**JIRA**: https://mobify.atlassian.net/secure/RapidBoard.jspa?rapidView=271&projectKey=WEB&modal=detail&selectedIssue=WEB-2067

## Changes
Pulled upstream from https://github.com/sintaxi/terraform
Address some other NPM vulnerabilities with npm audit and following the prompts

## How to test-drive this PR
- Change npm version to 6.0.0 via npm install -g npm@6.0.0
- Run npm audit
- See that there is the following vulnerability:
[!] 1 vulnerability found - Packages audited: 904 (28 dev, 25 optional)
Severity: 1 low

This one low vulnerability is resulting from:
terraform@1.7.0 /Users/lansichu/terraform
├── cli@1.0.1
└─┬ harp-minify@0.4.0
└─┬ html-minifier@1.1.1
└── cli@0.11.3


This project is a dependency for another project (mobify/harp) and that project is being used in as a dependency in documentation-theme.
